src/OpenLoco/src/Windows/Main.cpp

```diff
@@ -9,6 +10,9 @@ using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Ui::Windows::Main
 {
+    static loco_global<bool, 0x00F2533F> _showingGridlines;
+    static loco_global<bool, 0x0112C2E1> _showingDirectionArrows;
```

Comment:
I think these can just be integrated. Only the toggle methods relied on these two boolean states, anyway.